styling/utilities.css

```diff
@@ -0,0 +1,28 @@
+*{
+    box-sizing: border-box;
+    margin:0;
+    padding:0;
+}
+
+a{
```

Comment:
Don't add css direct to the tags. Give class instead

Reply:
done ...pls check now